src/DynamoCoreWpf/Controls/ShortcutToolbar.xaml.cs

```diff
@@ -34,21 +37,27 @@ public ObservableCollection<ShortcutBarItem> ShortcutBarRightSideItems
         {
             get { return shortcutBarRightSideItems; }
         }
+        private readonly DynamoModel _viewModel;
+        public ICommand SignOutCommand { get; private set; }
 
         /// <summary>
         /// Construct a ShortcutToolbar.
         /// </summary>
         /// <param name="updateManager"></param>
-        public ShortcutToolbar(IUpdateManager updateManager)
+        public ShortcutToolbar(DynamoModel dm)
```

Comment:
I only have concern for this known there will be conflict with @filipeotero 's https://github.com/DynamoDS/Dynamo/pull/13550